pkg/kubectl/cmd/util/factory.go

```diff
@@ -353,7 +354,7 @@ func (c *clientSwaggerSchema) ValidateBytes(data []byte) error {
 	//       instead of trying everything.
 	err = getSchemaAndValidate(c.c.RESTClient, data, "api", version, c.cacheDir)
 	if err != nil && c.ec != nil {
-		errExp := getSchemaAndValidate(c.ec.RESTClient, data, "experimental", version, c.cacheDir)
+		errExp := getSchemaAndValidate(c.ec.RESTClient, data, "apis"+"/"+latest.GroupOrDie("experimental").Group, version, c.cacheDir)
```

Comment:
@lavalamp, I'm pondering if using package `latest` in kubectl makes sense at all. In our discussion this afternoon, kubectl should retrieve the resource -> (resource, group) map from the API server, rather than using local one. We can apply the argument to package `latest` as well: kubectl should retrieve its group meta from API server, rather than using `latest` package compiled in the binary.